modules/swagger-codegen/src/main/resources/qt5cpp/helpers-body.mustache

```diff
@@ -132,64 +132,113 @@ setValue(void* value, QJsonValue obj, QString type, QString complexType) {
     }
     else if(type.startsWith("QList") && QString("").compare(complexType) != 0 && obj.isArray()) {
         // list of values
-        QList<void*>* output = new QList<void*>();
-        QJsonArray arr = obj.toArray();
-        foreach (const QJsonValue & jval, arr) {
-            if(complexType.startsWith("SWG")) {
+        if(complexType.startsWith("SWG")) {
+            QList<SWGObject*>* output = new QList<SWGObject*>();
+            QJsonArray arr = obj.toArray();
+            foreach (const QJsonValue & jval, arr) {
```

Comment:
why not use for (const QJsonValue & jval : arr)

Reply:
Done